src/cohort.js

```diff
@@ -0,0 +1,21 @@
+class Cohort {
+  constructor(name) {
+    this.students = []
+    this.CohortName = name
```

Comment:
`Cohort.CohortName` is a little redundant, `Cohort.name` would be a better choice here